### ISSUE TITLE
High CPU usage after mouseover

### ISSUE THREAD

**Issue by moreati** [cezarsa/silver_bird#52]
I'm using Chromed Bird on Chromium 4.0.276.0 (34953)/Ubuntu 9.10. I see 100% CPU usage on one core from when I mouseover the tweets area, until I close the Chromed Bird panel.

**Comment by moreati** [comment]
Hmm, something more subtle is happening. Chromed Bird doesn't always exhibit this behaviour. If a text box on the page of the current tab has keyboard focus when the CB icon is clicked, then the high CPU doesn't occur.

**Comment by moreati** [comment]
I'm no longer experiencing this problem. Whether due to a Chromium upgrade (currently 4.0.294.0 (35863) Ubuntu) or a Chromed Bird (1.4.2) upgrade I don't know.

**Comment by cezarsa** [comment]
Great, it probably wasn't anything I did.
There seem to be some still some issues with extensions on Chrome. Memory consumption is way too high (and keeps increasing as I repeatedly click the icon), and also is CPU usage when I start scrolling too much inside the popup.
For both these problems I really couldn't find any reasonable explanation, so I'm guessing there's something wrong with Chrome.
Closing this issue for now as it's already working fine for you.
